src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,13 @@
-import React from 'react';
+import { Movie } from '../../types/Movie';
```

Comment:
you don't have Movie file on types

Reply:
@AlinaMatuschak I've checked - all files were synced. Now it's fixed.